README.md

```diff
@@ -367,7 +367,7 @@ copycat.phoneNumber('foo')
 ```
 ```js
 copycat.phoneNumber('foo', { prefixes: ['+3319900', '+3363998'], min: 1000, max: 9999 })
-// => '+33639987662'
```

Comment:
**issue**

I think this should be changed to use `minLength: 4, maxLength: 4` instead.

Reply:
Yes indeed, thank you!